profiles/kentik_snmp/cisco/cisco-nexus.yml

```diff
@@ -111,3 +111,35 @@ metrics:
       name: cseSysMemoryUtilization
       poll_time_sec: 60
       tag: MemoryUtilization
+  - MIB: BGP4-MIB
+    symbols:
+    - name: cbgpPeer3State
+      OID: 1.3.6.1.4.1.9.9.187.1.2.9.1.5
+      enum:
+          idle: 1
+          connect: 2
+          active: 3
+          opensent: 4
+          openconfirm: 5
+          established: 6
+    - name: cbgpPeer3Type
+      OID: 1.3.6.1.4.1.9.9.187.1.2.9.1.2
+      enum:
+          unknown: 0
+          ipv4: 1
+          ipv6: 2
+          ipv4z: 3
+          ipv6z: 4
+          dns: 16
+    metric_tags:
+    - column:
+      - name: cbgpPeer3VrfId	
+        OID: 1.3.6.1.4.1.9.9.187.1.2.9.1.1
+    - column:
+      - name: cbgpPeer3RemoteAddr
+        OID: 1.3.6.1.4.1.9.9.187.1.2.9.1.3
+        conversion: hextoip
+    - column:
+      - name: cbgpPeer3VrfName
+        OID: 1.3.6.1.4.1.9.9.187.1.2.9.1.4
```

Comment:
this should be a tabular collection from the `cbgpPeer3Table` in the MIB. `cbgpPeer3State` would be the only metric and the other 4 OIDs should be under `metric_tags`* 


*including `cbgpPeer3Type`, this is a non-volatile metric and should be fine as a decoration on the status (we update the `metric_tag` values every 30min)